nixos/modules/programs/pqos-wrapper.nix

```diff
@@ -0,0 +1,27 @@
+{ config
+, lib
+, pkgs
+, ...
+}: {
+  options.programs.pqos-wrapper = {
+    enable = lib.mkEnableOption "PQoS Wrapper for BenchExec";
+    package = lib.mkPackageOption pkgs "pqos-wrapper" { };
+  };
+
+  config =
+    let
+      cfg = config.programs.pqos-wrapper;
+    in
```

Comment:
Normally we place this let in straight after the argument list at the top, so before options and config.

Reply:
Fixed in 1dd4d2283e09